README.md

```diff
@@ -9,6 +9,11 @@
 Please report any bugs related to Ubuntu Desktop Provision on [Launchpad](https://bugs.launchpad.net/ubuntu-desktop-provision).
 We use the GitHub issue tracker only for issues related to the development of Ubuntu Desktop Provision itself.
 
+## 24.04.1 OEM provisioning
+
+For configuring an OEM provisioning flow for Ubuntu 24.04.1 LTS, where user creation and account setup is handled
+seperately to system setup, please refer to [this guide](oem-provisioning-24_04_1.md).
```

Comment:
Should we maybe have all the docs in a docs directory, I'm guessing there will be more of them coming?

Reply:
Yea, we are headed that way. Other than this doc for now, do you think we should also move the CONTRIBUTING.md in there or is it more idiomatic to keep next to the README?